internal/btf/btf.go

```diff
@@ -176,16 +200,52 @@ func LoadKernelSpec() (*Spec, error) {
 }
 
 func loadKernelSpec() (*Spec, error) {
-	fh, err := os.Open("/sys/kernel/btf/vmlinux")
-	if os.IsNotExist(err) {
-		return nil, fmt.Errorf("can't open kernel BTF at /sys/kernel/btf/vmlinux: %w", ErrNotFound)
-	}
+	release, err := unix.KernelRelease()
 	if err != nil {
-		return nil, fmt.Errorf("can't read kernel BTF: %s", err)
+		return nil, err
+	}
+
+	locations := []struct {
+		pathFmt string
+		raw     bool
+	}{
+		/* try canonical vmlinux BTF through sysfs first */
```

Comment:
This list is from libbpf? Can you add a link to the source?

Reply:
Added